scripts/docker/build.gradle

```diff
@@ -0,0 +1,174 @@
+//Note: this section 'buildscript` is only for the dependencies of the buildscript itself.
+// See the second 'repositories' section below for the actual dependencies of GATK itself
+buildscript {
```

Comment:
I would probably rename this to something other than build.gradle  Maybe dockertest.gradle?